docroot/profiles/custom/su_humsci_profile/su_humsci_profile.info.yml

```diff
@@ -21,18 +27,23 @@ dependencies:
   - ckeditor
   - color_field
   - components
+  - config_readonly
```

Comment:
ooo nice.

Reply:
_if_ it works as i hope. when we get into multi site i plan to make the config that is included in a module read only. so they can still clone views if needed

Reply:
I would like this to work. My experience with the Earth team has taught me that the temptation is too great and stuff keeps dropping in to config ignore.